tools/topology/topology2/include/components/src_format.conf

```diff
@@ -1,369 +1,240 @@
+					in_valid_bit_depth      32
+				}
+				# 44.1khz input
+				{
+					in_rate                 44100
+					in_bit_depth            16
+					in_valid_bit_depth      16
+				}
+				{
+					in_rate                 44100
+					in_bit_depth            32
+					in_valid_bit_depth      24
+				}
+				{
+					in_rate                 44100
+					in_bit_depth            32
+					in_valid_bit_depth      32
+				}
+				# 48khz input
+				{
+					in_rate                 48000
+					in_bit_depth            16
+					in_valid_bit_depth      16
+				}
+				{
+					in_rate                 48000
+					in_bit_depth            32
+					in_valid_bit_depth      24
+				}
+				{
+					in_rate                 48000
+					in_bit_depth            32
+					in_valid_bit_depth      32
+				}
+				# 64khz input
+				{
+					in_rate                 64000
+					in_bit_depth            16
+					in_valid_bit_depth      16
+				}
+				{
+					in_rate                 64000
+					in_bit_depth            32
+					in_valid_bit_depth      24
+				}
+				{
+					in_rate                 64000
+					in_bit_depth            32
+					in_valid_bit_depth      32
+				}
+				# 88.2khz input
+				{
+					in_rate                 88200
+					in_bit_depth            16
+					in_valid_bit_depth      16
+				}
+				{
+					in_rate                 88200
+					in_bit_depth            32
+					in_valid_bit_depth      24
+				}
+				{
+					in_rate                 88200
+					in_bit_depth            32
+					in_valid_bit_depth      32
+				}
+				# 96khz input
+				{
+					in_rate                 96000
+					in_bit_depth            16
+					in_valid_bit_depth      16
+				}
+				{
+					in_rate                 96000
+					in_bit_depth            32
+					in_valid_bit_depth      24
+				}
+				{
+					in_rate                 96000
+					in_bit_depth            32
+					in_valid_bit_depth      32
+				}
+				# 176.4khz input
+				{
+					in_rate                 176400
+					in_bit_depth            16
+					in_valid_bit_depth      16
+				}
+				{
+					in_rate                 176400
+					in_bit_depth            32
+					in_valid_bit_depth      24
+				}
+				{
+					in_rate                 176400
+					in_bit_depth            32
+					in_valid_bit_depth      32
+				}
+				# 192khz input
+				{
+					in_rate                 192000
+					in_bit_depth            16
+					in_valid_bit_depth      16
+				}
+				{
+					in_rate                 192000
+					in_bit_depth            32
+					in_valid_bit_depth      24
+				}
+				{
+					in_rate                 192000
+					in_bit_depth            32
+					in_valid_bit_depth      32
+				}
+			]
+
+			num_output_audio_formats 1
+
+			Object.Base.output_audio_format [
+				{
+					out_rate                 48000
+					out_bit_depth            32
+					out_valid_bit_depth      32
+					# for 11.025k, 22.05k, 44.1k, 88.2k and 176.4k, extra 4 sample size is needed
+					obs "$[($out_channels * (($[($out_rate + 999)] / 1000) + 4)) * ($out_bit_depth / 8)]"
```

Comment:
but this is 48k?

Reply:
right, so this can be simplified now

Reply:
In conversion from some 44.1 -> 48 kHz the internal block sizes are not multiple of 1ms so I kept this. But in a quick test it ran also OK with with +0 instead of +4. If it looks safe to omit with more tests I'll do it. It also helps to get more 2^N like buffers sizes than might help with allocation.

Reply:
I'm removing this like, seems it's not needed.